### ISSUE TITLE
jpeg attachment type is not working

### ISSUE THREAD

**Issue by dcangulo** [chirag04/react-native-mail#147]
I am using [ivpusic/react-native-image-crop-picker ](https://github.com/ivpusic/react-native-image-crop-picker) for the attachments.

It returns `path` that works with this package, and it also returns `mime` that has a format like `image/jpeg`.

And to convert `image/jpeg` to file extension, I use [broofa/node-mime](https://github.com/broofa/node-mime).

The problem is that the following code does not work.
```javascript
attachment: {
  // other attributes
  type: mime.getExtension(image.mime) // this returns `jpeg`
}
```

Because `mime.getExtension(image.mime)` returns `jpeg` instead of `jpg`. I don't want to hardcode `jpg` for attachment type as I might have other types of images so instead, I rely on the `mime/type` to get the file extension.

**TL;DR**
`jpg` works but `jpeg` doesn't
